article/app/controllers/ArticleController.scala

```diff
@@ -128,20 +128,23 @@ class ArticleController extends Controller with RendersItemResponse with Logging
       renderFormat(htmlResponse, jsonResponse, article, Switches.all)
   }
 
-  def renderLiveBlog(path: String, page: Option[String] = None) =
-    Action.async { implicit request =>
+  def renderLiveBlog(path: String, page: Option[String] = None, format: Option[String] = None) =
+    if (format.contains("email"))
+      renderArticle(path)
```

Comment:
if format isn't email, you should throw a 404 ideally, so we don't cache the same content on more than one url